fetch/metadata/redirect/redirect-http-upgrade.sub.html

```diff
@@ -7,8 +7,8 @@
 <script src=/common/security-features/resources/common.sub.js></script>
 <script src=/common/utils.js></script>
 <body>
-    <script>
-  let nonce = "{{$id}}";
+<script>
+  let nonce = token();
```

Comment:
I don't think `nonce` is used anywhere in this file now (nor in `multiple-redirect-https-downgrade-upgrade.sub.html`). Is it used as a global from within `RunCommonRedirectTests()`?

Reply:
Yes:

https://github.com/web-platform-tests/wpt/blob/master/fetch/metadata/resources/redirectTestHelper.sub.js#L74

That's not great, but this PR doesn't seem like the best place to fix it.

Reply:
Got it. In that case, I agree!